### PR TITLE
fix(signpost): adjust trigger pointer positions

### PR DESCRIPTION
**Pull request by valentin-mladenov**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [ ] If applicable, have a visual design approval

## PR Type

What kind of change does this PR introduce?

- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: CDE-818

## Does this PR introduce a breaking change?

- [x] Yes
- [ ] No

BREAKING CHANGE: Signpost position offsets are now changed to match design spec.

### CODE REVIEW

**Comment by valentin-mladenov** [comment]
depends on: #810

File: projects/angular/src/button/_buttons.clarity.scss [hunk 1]
```diff
@@ -380,7 +380,9 @@
   }
 
   .btn.btn-sm,
-  .btn-sm .btn {
+  .btn-sm .btn,
+  .btn.btn-small,
+  .btn-small .btn {
```
Comment (dtsanevmw) [inline projects/angular/src/button/_buttons.clarity.scss:385]: We don't need to add `btn-small` as class in the buttons, just fix the class on the signpost-button to be `btn-sm`

**Review by dtsanevmw — CHANGES_REQUESTED** [review]
Change the class `btn-small` to `btn-sm` on signpost button.

File: projects/angular/src/button/_buttons.clarity.scss (hunk 1 above)
Comment (valentin-mladenov) [reply projects/angular/src/button/_buttons.clarity.scss:385]: We have docs for `btn-small`, but not for `btn-sm`. Decided to retain documentation intact and all other use cases. It's still work in progress.

Comment (dtsanevmw) [reply projects/angular/src/button/_buttons.clarity.scss:385]: It should be `btn-sm` in that `STYLES.md` file too. It's `btn-sm` in all other components and also on our website. Adding a second thing which is wrong will just make it more confusing.

Comment (valentin-mladenov) [reply projects/angular/src/button/_buttons.clarity.scss:385]: Well not all other components. Datagrid is using `btn-small`, but full migration to `btn-sm` is the right way. I'll make the change.

Comment (valentin-mladenov) [reply projects/angular/src/button/_buttons.clarity.scss:431]: Probably should be best to move renaming to another PR.

Comment (valentin-mladenov) [reply projects/angular/src/button/_buttons.clarity.scss:431]: PR for `btn-small` created #912

**Comment by valentin-mladenov** [comment]
https://github.com/vmware-clarity/ng-clarity/pull/848#pullrequestreview-1556755123

@dtsanevmw Can't replay to your comment directly. Your request is merged with #912

**Review by dtsanevmw — APPROVED** [review]

**Pull request merged**